Tests fail to build after commit b795d0da5b274ad0da5d52d5e45a98f21521007c

Hi! The latest commit broke tests :/

```text
[ 62%] Building CXX object src/test_verifier/CMakeFiles/test_verifier.dir/main.cpp.o
cd obj-x86_64-linux-gnu/src/test_verifier && /usr/bin/c++  -I../src/test_verifier/. -I../include -g -O2 -std=c++17  -Wall -Wextra -Wcast-qual -pedantic -pedantic-errors -Wfatal-errors -Wno-error=maybe-uninitialized -Wno-missing-braces -Wno-unused-parameter -Werror -MD -MT src/test_verifier/CMakeFiles/test_verifier.dir/main.cpp.o -MF CMakeFiles/test_verifier.dir/main.cpp.o.d -o CMakeFiles/test_verifier.dir/main.cpp.o -c ../src/test_verifier/main.cpp
In file included from ../src/test_verifier/main.cpp:7:
../src/test_verifier/../interpreter.h:15:10: fatal error: register.h: No such file or directory
   15 | #include "register.h"
      |          ^~~~~~~~~~~~
compilation terminated.
```

--- a/src/test_verifier/CMakeLists.txt
+++ b/src/test_verifier/CMakeLists.txt
@@ -6,6 +6,7 @@ add_executable(test_verifier
 create_target_directory_groups(test_verifier)
 target_link_libraries(test_verifier PRIVATE teakra)
 target_include_directories(test_verifier PRIVATE .)
+target_include_directories(test_verifier PRIVATE ../../include/teakra/impl/)
 target_compile_options(test_verifier PRIVATE ${TEAKRA_CXX_FLAGS})
 
 

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -6,6 +6,7 @@ add_executable(teakra_tests
     timer.cpp
 )
 
+target_include_directories(teakra_tests PRIVATE ../include/teakra/impl/)
 target_link_libraries(teakra_tests PRIVATE teakra catch)
 target_compile_options(teakra_tests PRIVATE ${TEAKRA_CXX_FLAGS})
 

